### PR TITLE
Siva | MOBN-619 | Add test-ids for conflicts components and update translation message to distinguish normal and recurring appointment

### PR DESCRIPTION
**Pull request by shivarach**

### CODE REVIEW

**Comment by shivarach** [comment]
The second commit updates the translation message if it is recurring.

- The appointment you are trying to book overlaps with the following dates

- The **recurring** appointments you are trying to book overlaps with the following dates

**Comment by shivarach** [comment]
The second commit updates a translation message

- The appointment you are trying to book overlaps with the following dates
- The **recurring appointments** you are trying to book overlaps with the following dates

File: ui/react-components/components/Conflicts/Body/ConflictsBody.jsx [hunk 1]
```diff
@@ -18,18 +18,18 @@ const ConflictsBody = props => {
...
-            tabHeads.push(createTab(serviceUnavailableConflicts, 'NO_SERVICE_DATE_CONFLICTS', 'No-Service Date conflicts'));
+            tabHeads.push(createTab(serviceUnavailableConflicts, 'NO_SERVICE_DATE_CONFLICTS', 'No-Service Date conflicts', 'noServiceDateConflicts'));
             tabPanels.push(createTabPanel(serviceUnavailableConflicts, SERVICE_UNAVAILABLE_CONTENT));
         }
         return {tabHeads, tabPanels};
     };
 
-    const createTab = (conflicts, translationKey, defaultMessage) => {
-        return <Tab key={defaultMessage}>
+    const createTab = (conflicts, translationKey, defaultMessage, dataTestId) => {
```
Comment (bsneha90) [inline ui/react-components/components/Conflicts/Body/ConflictsBody.jsx]: Why do we need different test id for same control? The content of the control should be enough for tests right?

File: ui/react-components/components/Conflicts/Body/ConflictsBody.jsx [hunk 2]
```diff
@@ -18,18 +18,18 @@ const ConflictsBody = props => {
...
+            tabHeads.push(createTab(serviceUnavailableConflicts, 'NO_SERVICE_DATE_CONFLICTS', 'No-Service Date conflicts'));
             tabPanels.push(createTabPanel(serviceUnavailableConflicts, SERVICE_UNAVAILABLE_CONTENT));
         }
         return {tabHeads, tabPanels};
     };
 
-    const createTab = (conflicts, translationKey, defaultMessage, dataTestId) => {
-        return <Tab key={defaultMessage} data-testid={dataTestId}>
+    const createTab = (conflicts, translationKey, defaultMessage) => {
+        return <Tab key={defaultMessage} data-testid={"conflict"}>
```
Comment (bsneha90) [inline ui/react-components/components/Conflicts/Body/ConflictsBody.jsx]: can it be called conflict-tab??

**Pull request merged**